internal/bzlmod/default_gazelle_overrides.bzl

```diff
@@ -15,67 +15,12 @@
 visibility("private")
 
 DEFAULT_BUILD_FILE_GENERATION_BY_PATH = {
-    "github.com/envoyproxy/protoc-gen-validate": "on",
-    "github.com/google/safetext": "on",
-    "github.com/grpc-ecosystem/grpc-gateway/v2": "on",
-    "google.golang.org/grpc": "on",
+    "github.com/grpc-ecosystem/grpc-gateway/v2": "update",
+    "github.com/bazelbuild/buildtools": "update",
```

Comment:
Hm, no, I think that's accidental. It should be `off`, I believe. Will test it to see which one it was 😄

Reply:
Ok, found it.

Any of `update`, `off`, or `clean` will work here in a 'vanilla' bazel repo. The reason I set it to `update` and `off` is that _my own code_ was importing from the buildtools repo, and since that was my test case I accidentally blindly copied that over. The reality is that the majority of users have no need for this and only have buildtools as a transient dep of gazelle and rules_go, so... I think it can go? 😄 

Will update my PR to remove this.